index.html

```diff
@@ -48,79 +55,79 @@
+		<textarea class="editbuttons tanksettings pos7" id="bodydamage" rows="1" cols="10" placeholder="0" data-tooltip="Specifies the damage the body of the tank does to a shape on collision. NOTE: Does not save. Yet.">50</textarea>
 		
-		<p class="editbuttons barrelsettings pos1label" id="traplabel"><b>Barrel </b><span title="Specifies what type of barrel you want to place." class="info">ⓘ</span></p>
-		<select class="editbuttons barrelsettings pos1" id="bullet" title="Specifies what type of barrel you want to place.">
+		<p class="editbuttons barrelsettings pos1label" id="traplabel"><b>Barrel </b><span data-tooltip="Specifies what type of barrel you want to place." class="info">ⓘ</span></p>
+		<select class="editbuttons barrelsettings pos1" id="bullet" data-tooltip="Specifies what type of barrel you want to place.">
 			<option value="bullet">Gun</option> 
   			<option value="trap">Trap Layer</option>
   			<option value="drone">Drone Maker</option>
 			<option value="necro">Sunchips!</option>
 		</select>
 		
-		<p class="editbuttons barrelsettings pos2label"><b>Can Shoot? </b><span title="Toggles this cannon's ability to shoot." class="info">ⓘ</span></p>
-		<input type="checkbox" class="editbuttons barrelsettings pos2" id="disable" checked title="Toggles this cannon's ability to shoot.">
+		<p class="editbuttons barrelsettings pos2label"><b>Can Shoot? </b><span data-tooltip="Toggles this cannon's ability to shoot." class="info">ⓘ</span></p>
+		<input type="checkbox" class="editbuttons barrelsettings pos2" id="disable" checked data-tooltip="Toggles this cannon's ability to shoot.">
 		
-		<p class="editbuttons barrelsettings pos3label"><b>Width </b><span title="Specifies the width of the cannon." class="info">ⓘ</span></p>
-		<textarea class="editbuttons barrelsettings pos3" id="width" rows="1" cols="10"  placeholder="0" title="Specifies the width of the cannon.">15</textarea>
+		<p class="editbuttons barrelsettings pos3label"><b>Width </b><span data-tooltip="Specifies the width of the cannon." class="info">ⓘ</span></p>
+		<textarea class="editbuttons barrelsettings pos3" id="width" rows="1" cols="10"  placeholder="0" data-tooltip="Specifies the width of the cannon.">15</textarea>
 		
-		<p class="editbuttons barrelsettings pos4label"><b>Length </b><span title="Specifies the length of the cannon." class="info">ⓘ</span></p>
-		<textarea class="editbuttons barrelsettings pos4" id="length" rows="1" cols="10"  placeholder="0" title="Specifies the length of the cannon.">60</textarea>
+		<p class="editbuttons barrelsettings pos4label"><b>Length </b><span data-tooltip="Specifies the length of the cannon." class="info">ⓘ</span></p>
+		<textarea class="editbuttons barrelsettings pos4" id="length" rows="1" cols="10"  placeholder="0" data-tooltip="Specifies the length of the cannon.">60</textarea>
```

Comment:
I say just merge the pull request for now. Not many people know that you can hover over the `<textarea>`s (including **me** until you pointed it out; the reason the `<textarea>`s had `data-tooltip` was because I did find-and-replace all) anyways. As long as the noscript and tooltip for `<span>` works, it's OK for now. I'll be looking into the problem with the `<textarea>` soon.